src/Pages/Questions/Questions.css

```diff
@@ -0,0 +1,83 @@
+.content-container{
+    height: 40rem;
+    width: 81rem;
+    display: flex;
+    justify-content: center;
+    border: 1px solid var(--black-color);
+}
+.outer-div{
+    height: 28rem;
+    width: 35rem;
+    display: flex;
+    justify-content: center;
+    margin-top: 2rem;
+    color: var(--white-color);
+    border-radius: 1rem;
+    box-shadow: 1px 1px 4px 3px var(--gainsboro-color);
+}
+.inner-div{
+    height: 30rem;
+    width:30rem ;
+    margin-top: 1rem;
+    display: flex;
+    justify-content: center;
+}
+.options{
+    position: relative;
+    top: 12rem;
+    right: 32.6rem;
+    height: 15rem;
+    width: 30rem;
+    text-align: center; 
+    
+}
+.Question-container{
+    height: 10rem;
+    width: 50rem;
+    text-align: center;    
+}
+.btn{
+    padding: 0.8rem 1rem;
+    background-color: var(--white-color);
+    position: relative;
+    top:5rem;
+    left: 14rem;
+    text-align: start;
+    color: var(--black-color);
+    border: none;
+    border-radius: 0.5rem;
+    font-size: 1rem;
+    cursor: pointer;
+    font-weight: 900;
+}
+span{
+    margin-left: 25rem;
+}
+.angleRight{
+    color:var(--dark-red);
+    position: absolute;
+}
+.selectedOption {
+    background-color: var(--gainsboro-color);
+}
+.each_option:hover {
+    cursor: pointer;
+    transform: scale(1.02);
+    background-color: var(--gainsboro-color);
+}
+.each_option {
+    border-radius: 0.5rem;
+    padding: 0.8rem 2rem;
+    border-radius: 0.5rem;
```

Comment:
try to give border-radius in px, to avoid change in card shape while changing screensize